### ISSUE TITLE
Cannot call function writeRelay() twice

### ISSUE THREAD

**Issue by JakesCustomShop** [SequentMicrosystems/Sequent-4rel4in-Library#1]
It seems as though two relays cannot be toggled at the same instance. 
`        IO_Card.writeRelay(GRN, HIGH);
         IO_Card.writeRelay(BUZZ, HIGH); `

Instead they need a 10mS delay between calls:
`        IO_Card.writeRelay(GRN, HIGH);       
        delay(10);
        IO_Card.writeRelay(BUZZ, HIGH); `

Not sure why this is

**Comment by alexburcea2877** [comment]
Yes you are correct, setting one relay at a time must be done with 10ms delay, but you have the second option to update all relays in the same time which can be done with the same function with a single parameter.  This single parameter represents the relays state as a bitmap.

**Comment by JakesCustomShop** [comment]
Cool, thanks.
